src/App.jsx

```diff
@@ -25,12 +26,10 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
+    <Person person={misha} />
+    <Person person={olya} />
+    <Person person={alex} />
+{/* 
```

Comment:
Remove this comments 
<img width="611" alt="image" src="https://github.com/user-attachments/assets/5c708705-c5b9-4a61-ba01-8f42ce7f4d9a">